src/database/modele/Tag.js

```diff
@@ -20,12 +22,31 @@ const TagSchema = new mongoose.Schema({
     //Multiple discussion and question can linked the same tag declaration of arrays
     discussions:[
         {
-            discussionTag: {type: Number, ref: 'Discussion'}
+            discussionTag: {
+                type: Number,
+                ref: 'Discussion',
+                //Validate will permit us to make some validation before a dicussionTag is saved
+                //The function check if the id of the value passed exits in the DataBase if yes it will return true otherwise false
```

Comment:
```suggestion
                // Validate will permit us to make some validation before a dicussionTag is saved
                // The function check if the id of the value passed exits in the DataBase if yes it will return true otherwise false
```